processor/src/services/operation.service.ts

```diff
@@ -56,7 +57,7 @@ export class DefaultOperationService implements OperationService {
 
     const transactionType = this.getPaymentTransactionType(request.action);
 
-    await this.ctPaymentService.updatePayment({
+    const payment = await this.ctPaymentService.updatePayment({
```

Comment:
I would rename this to `updatedPayment` or `modifiedPayment` as we got previously a reference to the payment object